### DISCUSSION TITLE
Factura 4.0

### DISCUSSION THREAD

**Discussion by jibleg** [bambucode/tfacturaelectronica#183] [General]
Al generar el CFDI 4.0 en el nodo Concepto y Traslado, agrega el namespace. 
```
  <cfdi:Conceptos>
    <Concepto xmlns="http://www.sat.gob.mx/cfd/4" ClaveProdServ="85122200" NoIdentificacion="1" Cantidad="1" ClaveUnidad="E48" Unidad="Servicio" Descripcion="ALCOHOL ETILICO EN SUERO (ETANOL)" ValorUnitario="251.00" Importe="251.00" Descuento="0.00" ObjetoImp="02">
      <Impuestos>
        <Traslados>
          <Traslado Base="251.00" Impuesto="002" TipoFactor="Tasa" TasaOCuota="0.160000" Importe="40.16"/>
        </Traslados>
      </Impuestos>
    </Concepto>
   </cfdi:Conceptos>
  <cfdi:Impuestos TotalImpuestosTrasladados="700.51">
    <cfdi:Traslados>
      <Traslado xmlns="http://www.sat.gob.mx/cfd/4" Base="4378.17" Impuesto="002" TipoFactor="Tasa" TasaOCuota="0.160000" Importe="700.5072"/>
    </cfdi:Traslados>
  </cfdi:Impuestos>

```
¿Alguna sugerencia para corregir la incidencia?

**Comment by lcarrasco** [comment]
¿Si estás probando con la versión mas reciente? En mi caso si genera los nodos hijo con el namespace correcto ...

**Comment by jibleg** [comment]
Hola Luis. Buenas tardes. Si baje el código a inicio del año. Igual bajaré el último publicado en github y hago la implementación para ver si soluciona mi inconveniente Sent from my iPhone On 24 Mar 2023, at 17:53, Luis Carrasco ***@***.***> wrote:﻿
¿Si estás probando con la versión mas reciente? En mi caso si genera los nodos hijo con el namespace correcto ...

**Comment by jibleg** [comment]
Aunque te comento. Mi PAC finkok en su validador me lanza el error derivado del fragmento de XML que comparti tiene en el nodo el namespace y ellos indican que no debe colocarse el namespace pues ya se coloca en el encabezado del XML Sent from my iPhone On 24 Mar 2023, at 17:53, Luis Carrasco ***@***.***> wrote:﻿
¿Si estás probando con la versión mas reciente? En mi caso si genera los nodos hijo con el namespace correcto ...

**Reply by jibleg** [reply]
Mi incidencia es la siguiente: En los nodos **Traslados**
```
        <cfdi:Traslados>
          <Traslado **xmlns="http://www.sat.gob.mx/cfd/4"** Base="1350.00" Impuesto="002" TipoFactor="Tasa" TasaOCuota="0.160000" Importe="216.00"/>
        </cfdi:Traslados>
```
se agrega por defecto el namespace xmlns="http://www.sat.gob.mx/cfd/4 lo que me produce el siguiente error:
 Validación de estructura: Prefijo y namespace con error: None - http://www.sat.gob.mx/cfd/4
Alguna sugerencia para resolverlo.

Saludos cordiales.
